utils/tangleid/TangleID.js

```diff
@@ -23,7 +23,6 @@ class TangleID {
     const namespace = extension.getNamespace();
     extension.setup(this.api);
```

Comment:
Why do we need to setup CoreAPI into extension?
The service provider should be different right?